### PR TITLE
Delete

### LINKED ISSUE #12: Create mutation for delete_phrase

### PR DESCRIPTION
**Pull request by EricLarson2020**
### Description
Added delete mutations for phrases and sections. 
Updated readme to include setup and schema

### Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which maintains existing functionality)
 - [yes] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,32 @@
...
+- `$ rake db:migrate`
+- `$ rake db:seed`
+
+Once this is all set up and you aren't getting any errors you can run our test suite.
+
+- `$ bundle exec rspec`
+
+### Schema
+
+![Schema](/public/gift_of_gab_schema.png)
```
Comment (cgaddis36) [inline README.md:32]: nice job adding the setup here, looks nice

File: app/graphql/mutations/phrases/delete_phrase.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+      def resolve(id:)
+        phrase = Phrase.find(id)
+          phrase.destroy!
+          { phrase: phrase }
+
+        end
+      end
+    end
+  end
```
Comment (cgaddis36) [inline app/graphql/mutations/phrases/delete_phrase.rb:16]: very clean here

File: spec/requests/v1/mutations/section/delete_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    GRAPHQL
+    post "/v1/graphql", params: { query: mutation_string }
+    expect(response.status).to eq(200)
+    parsed_response = JSON.parse(response.body, symbolize_names: true)
+
+    expect(parsed_response[:data][:deleteSection][:section]).to have_key(:id)
+    expect(parsed_response[:data][:deleteSection][:section]).to have_key(:title)
+    expect(parsed_response[:data][:deleteSection][:section]).to have_key(:icon)
+    expect(Section.all).to eq([])
+
```
Comment (cgaddis36) [inline spec/requests/v1/mutations/section/delete_spec.rb:29]: nice thorough testing

**Review by cgaddis36 — APPROVED** [review]

File: app/graphql/types/mutation_type.rb [hunk 4]
```diff
@@ -2,5 +2,7 @@ module Types
   class MutationType < Types::BaseObject
     field :updateUser, mutation: Mutations::Users::UpdateUser
     field :newSection, mutation: Mutations::Sections::NewSection
+    field :deletePhrase, mutation: Mutations::Phrases::DeletePhrase
```
Comment (sagemlee) [inline app/graphql/types/mutation_type.rb:5]: Nice, way to knock these out.

**Review by sagemlee — APPROVED** [review]

**Pull request merged**